crowbar_framework/app/models/api/node.rb

```diff
@@ -72,7 +69,26 @@ def os_upgrade
         Rails.logger.error("Executing of os upgrade script has failed on node #{@node.name}.")
         return false
       end
-      true
+
+      begin
```

Comment:
The problem is with those failure / success messages. I actually want to not only log a messages, but (in future) use a status library to update the current status of the upgrade. And I did not want to put such upgrade library call to the old `node_object.rb`